File: ParadigmasdaProgramacao/atividadeJava/src/Questoes/Aluno.java
package Questoes;

public class Aluno {

    private String nomeAluno;
    private Double primeiraNota;
    private Double segundaNota;
    private Double terceiraNota;

    public Aluno(){
    }

    public static void  calcularMediaAluno (Double primeiraNota, Double segundaNota, Double terceiraNota){
       double notaFinal = (primeiraNota + segundaNota + terceiraNota) ;
        if ( notaFinal >= 60 ){
            System.out.println("Aluno Aprovado");
        } else{
            System.out.println("Aluno Reprovado");
            System.out.println("Faltam: " + (60 - notaFinal));
        }
    }

    public String getNomeAluno() {
        return nomeAluno;
    }

    public void setNomeAluno(String nomeAluno) {
        this.nomeAluno = nomeAluno;
    }

    public Double getPrimeiraNota() {
        return primeiraNota;
    }

    public void setPrimeiraNota(Double primeiraNota) {
        this.primeiraNota = primeiraNota;
    }

    public Double getSegundaNota() {
        return segundaNota;
    }

    public void setSegundaNota(Double segundaNota) {
        this.segundaNota = segundaNota;
    }

    public Double getTerceiraNota() {
        return terceiraNota;
    }

    public void setTerceiraNota(Double terceiraNota) {
        this.terceiraNota = terceiraNota;
    }

}

File: ParadigmasdaProgramacao/atividadeCsharp/atividadeCsharp/Aluno.cs
using System;

namespace atividadeCsharp
{

    public class Aluno
    {
        public Aluno()
        {
        }

        private string nomeAluno;

        private double primeiraNota;

        private double segundaNota;

        private double terceiraNota;

        private double notaFinal;

        private string Resultado;

        public void CalcularMediaAluno(double primeiraNota, double segundaNota, double terceiraNota)
        {
            notaFinal = this.primeiraNota + this.segundaNota + this.terceiraNota;

            if (notaFinal >= 60)
            {
                Console.WriteLine("Aluno Aprovado");
            }
            else
            {
                Console.WriteLine("Aluno Reprovado");
                Console.WriteLine("Faltam: " + (60 - notaFinal));
            }
        }

        public string GetNomeAluno()
        {
            return nomeAluno;
        }

        public void SetNomeAluno(string nomeAluno)
        {
            this.nomeAluno = nomeAluno;
        }

        public double GetPrimeiraNota()
        {
            return primeiraNota;
        }

        public void SetPrimeiraNota(double primeiraNota)
        {
            this.primeiraNota = primeiraNota;
        }

        public double GetSegundaNota()
        {
            return segundaNota;
        }

        public void SetSegundaNota(double segundaNota)
        {
            this.segundaNota = segundaNota;
        }

        public double GetTerceiraNota()
        {
            return terceiraNota;
        }

        public void SetTerceiraNota(double terceiraNota)
        {
            this.terceiraNota = terceiraNota;
        }

        public double GetNotaFinal()
        {
            return notaFinal;
        }

        public void SetNotaFinal(double notaFinal)
        {
            this.notaFinal = notaFinal;
        }

        public string GetResultado()
        {
            return Resultado;
        }

        public void SetResultado(string resultado)
        {
            Resultado = resultado;
        }
    }
}